training.md

```diff
@@ -36,45 +36,122 @@ There may be an issue where your terminal can't find Git or Anaconda, in which c
+6. Let's verify that we can connect to the database successfully.
+   1. Run `dbt debug`. This command tries to connect to the database using the parameters from `profiles.yml` and `dbt_project.yml`.
+      * You connected successfully if you see all green and no error messages!
+
+### Azure Data Studio
+
+1. Last step in the set-up is to log in to the database
+   1. Open Azure Data Studio and click on the first icon on the left side panel called "Connections".
+   2. Click "Add Connection" and paste in the connection info below in the relevant fields. This info can also be found in `profiles.yml`
+      * Server: dbtavatrain.database.windows.net
+      * Authentication type: Azure Active Directory
+      * Account: Click the drop down menu, then "Add an account" and sign in like before
+      * Azure AD tenant: Avanade
+      * Database: Manually type "sandbox"
+   3. Click Connect
+      1. You are now connected to the database! 
+         * Please note that there may be many tables in the database already. These are other people's tables that they created using their schema name. Yours will start to show up as you start creating your dbt models.
+2. Now you are ready to start the exercises in the next section below! 
+   * If you are not familiar with Azure Data Studio, here are some pro tips that might be useful when going through the exercises:
+      1. The server you are connected to and the **Tables** and **Views** folders are located on the left side of the page
+         1. To quickly query a table or view: Go into the relevant folder, right click the name, and click "Select Top 1000".
+         2. To write a new query, right click the server, and click "New Query"
+      3. If you're wondering why you're not seeing your tables/views show up after running dbt commands in VSCode, you can right click on the **Tables** or **Views** folder and click "Refresh".
+         * You can also do this to the server itself as well, but this might take longer.
+
+
+## 2. Learning dbt
+
+Try the walk-through tutorial of using dbt below. There are also additional resources found at the end of this section as well. Happy modeling!
+
+### Tutorial
+
+#### 0. Connect to External Data Source
+Before we jump in and start creating dbt models, we need to first connect to our external data source and pull our source tables from there. We will be pulling data from a blob storage called `JaffleShopBlob`.
+
+1. Open Azure Data Studio and create a "New Query" by right clicking on the server name and choosing the option.
+2. Paste the Azure SQL code below into the new query and click "Run" at the top. To view the Synapse version of the code click [here](external_synapse.sql).
+   * *should we also have them edit the name of the table? bc everyone will have raw.customers already and there will be an error message when querying*
```

Comment:
can you refresh my memory if you want everyone going through the tutorial to be able to run this code, or if you just want them to look at it and understand how we're connecting to the external data source?

Reply:
if you're making your own db and not using a db that has been made for them, then they'll need to run this script to create the three raw tables